introduction.md

```diff
@@ -1 +1,5 @@
-Lol, hi! JavaScript is my favorite path.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?